sources/packages/web/src/composables/institution/useInstitutionAuth.ts

```diff
@@ -1,34 +1,60 @@
 import {
   InstitutionUserAuthRolesAndLocation,
   InstitutionUserRoles,
+  UserStateForStore,
 } from "@/types";
 import { computed } from "vue";
-import { useStore } from "vuex";
+import { Store, useStore } from "vuex";
 import { useAuth } from "..";
 
-export function useInstitutionAuth() {
-  const store = useStore();
+export function useInstitutionAuth(rootStore?: Store<any>) {
+  const store = rootStore ?? useStore();
+
+  // Use store getters to get user details and authorizations.
+  const institutionUserDetails = store.getters[
```

Comment:
Looking at the getters as they are I do not see a benefit since they are not actually transforming or doing any operation on the state.
IMO we can even get rid of the getters and use the state in this composable as it was before, for instance,
using `store.state.institution.userState`, `store.state.institution.authorizationsState`.